lib/sass_spec/test.rb

```diff
@@ -1,9 +1,16 @@
 require 'minitest'
 
+parallelize = false
+
 def run_spec_test(test_case, options = {})
   if options[:skip_todo] && test_case.todo?
     skip "Skipped todo"
   end
+  # When running executables we can safely parallelize
+  # Executables must become their own process environment
+  if options[:engine_adapter].instance_of? (ExecutableEngineAdapater)
```

Comment:
Instances should have a paralellizable? method that determines if it can be parallelized or not.

That's also the reason why our use of RTTI is bad, but I digress...